oda_api/api.py

```diff
@@ -837,7 +837,7 @@ def get_product(self,
         return d
 
     @staticmethod
-    def set_api_code(query_dict):
+    def set_api_code(query_dict, url="www.astro.unige.ch/cdci/astrooda/dispatch-data"):
```

Comment:
so how will it look like? With tesT?

Reply:
on it, but on a PR on the dispatcher-app

Reply:
could be here too with dispatcher-fixture. depends which code is tested. as you like

Reply:
actrually is faster

Reply:
ok!
in principle, would be good to also make API-focused tests with fixture here: https://github.com/oda-hub/oda_api/issues/13
but really after